external-crates/move/crates/move-vm-types/src/values/values_impl.rs

```diff
@@ -1556,6 +1556,26 @@ impl VMValueCast<Vec<u8>> for Value {
     }
 }
 
+impl VMValueCast<Vec<u16>> for Value {
```

Comment:
what are those functions for? sorry I am a bit confused here

Reply:
Casting from a primitive of a `vector<u16>` to `Vec<u16>`